### DISCUSSION TITLE
nocodb with Azure MySQL [Apr 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1920] [Discord Threads]
> 🙋‍♂️ **blakshmi**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### could you please share how you host nocodb?  
#  
> 🙋‍♂️ **blakshmi**  
#### When I'm connecting like this showing time out  
![image0](https://cdn.discordapp.com/attachments/965947323345670244/965964978580500520/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/965947323345670244/965964978790228018/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### please check your firewall rules  
#  
> 👨‍🚀 **wingkwong**  
#### check if the traffic is allowed from nocodb instance  
#  
> 🙋‍♂️ **blakshmi**  
#### ok I'll check it  
#  
> 🙋‍♂️ **blakshmi**  
#### Hi can we quickly connect through google meet?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965947323345670244)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
